converter/internal/prometheusconvert/testdata/file.river

```diff
@@ -0,0 +1,43 @@
+discovery.file "prometheus1" {
+	files = ["/tmp/example_*.yaml", "/tmp/example2_*.yaml"]
+}
+
+discovery.file "prometheus2" {
+	files            = ["/tmp/example_*.yaml", "/tmp/example2_*.yaml"]
+	refresh_interval = "1m0s"
+}
+
+prometheus.scrape "prometheus1" {
+	targets = concat(
+		discovery.file.prometheus1.targets,
+		[{
+			__address__ = "localhost:9090",
+		}],
+	)
+	forward_to = [prometheus.remote_write.default.receiver]
+	job_name   = "prometheus1"
+}
+
+prometheus.scrape "prometheus2" {
+	targets    = discovery.file.prometheus2.targets
+	forward_to = [prometheus.remote_write.default.receiver]
+	job_name   = "prometheus2"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name           = "remote1"
+		url            = "http://remote-write-url1"
+		send_exemplars = false
+
+		queue_config {
```

Comment:
@thampiotr we are in a race here with your PR updating the prometheus version with an unblocking merge conflict for this portion of the test.